pkg/docscollector/docscollector.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docscollector
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+	"text/template"
+)
+
+//Docsdb is struct to store the decoded docs metadata
+type Docsdb struct {
+	MetricName    string `json:"MetricName"`
```

Comment:
This is more or less a https://github.com/kubernetes/kube-state-metrics/blob/04f2249bdb8464109f073d2cd114db23090c2cbe/pkg/metric_generator/generator.go#L30-L36 struct already no? I think we could reuse that with maybe some small modifications to it?

Reply:
Working on this